library/alloc/src/collections/btree/node.rs

```diff
@@ -99,15 +106,20 @@ struct InternalNode<K, V> {
 }
 
 impl<K, V> InternalNode<K, V> {
-    /// Creates a new `InternalNode`.
+    /// Creates a new boxed `InternalNode`.
     ///
-    /// This is unsafe for two reasons. First, it returns an `InternalNode` by value, risking
+    /// This is unsafe for two reasons. First, it returns an owned `InternalNode` in a box, risking
```

Comment:
I don't understand this unsafety comment, for both leaf and internal nodes. None of the fields are able to drop anything, keys and values are shielded by `MaybeUninit`, and the node module does not define any drop handler and explicitly says so.

Reply:
I think it's just outdated/not written with safety really correctly understood.